readme.txt

```diff
@@ -120,11 +120,8 @@ All is not lost!
 
 == Changelog ==
 
-**v3.11.2**
```

Comment:
this supposed to be removed?

Reply:
I just add the latest one with a link to the main list